src/ensq_channel.erl

```diff
@@ -122,7 +122,7 @@ loop(State) ->
                              %% We don't want to ask for a propper new RC every time
```

Comment:
Line 122 is too long:                              %% We don't want to ask for a propper new RC every time.